tcs-service/src/test/java/com/transformuk/hee/tis/tcs/service/service/impl/ConditionsOfJoiningServiceImplTest.java

```diff
@@ -134,6 +134,27 @@ void saveShouldSaveTheConditionsOfJoiningAgainstTheProgrammeMembershipFromPmId()
         is(GoldGuideVersion.GG9));
   }
 
+  @Test
```

Comment:
This could be combined with the above test by using a parameterised test with enum source